### PR TITLE
Add Reolink chime docs

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add docs for the Reolink chime entities, see https://github.com/home-assistant/core/pull/122752

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122752
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced customizable chime settings for the Reolink camera integration, allowing users to adjust chime volume and select from various ringtones.
	- Added new settings for different alert types including motion, person, and visitor notifications.
	- Implemented a "Chime LED" option for controlling the visual indicator associated with chimes.

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -168,6 +169,10 @@ Depending on the supported features of the camera, select entities are added for
 - Auto track method (Digital, Digital first, Pan/Tilt first)
 - Doorbell LED (Stay off, Auto, Auto & always on at night)
 - HDR* (Off, On, Auto)
+- Chime play ringtone
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]: what does the "loop" option do? loop over a default ringtone? Or is Loop the name of a ringtone?

Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: Loop is just the name of a ringtone.
I did not think of the names, they come from the Reolink app

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @starkillerOG :+1:

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (edenhaus) [inline source/_integrations/reolink.markdown]:
```suggestion
```
As it was removed from the PR

**Review by edenhaus — CHANGES_REQUESTED** [review]

**Review by edenhaus — APPROVED** [review]

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/reolink.markdown]: thank you, @edenhaus !

**Pull request merged**